CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ include(${Slicer_USE_FILE})
 add_subdirectory(BreachWarningLight)
 add_subdirectory(Viewpoint)
 add_subdirectory(LumpNav)
+add_subdirectory(TipToSurfaceDistance)

```

Comment:
Well I guess I should have looked closer before I implemented the TipToSurfaceDistance module but fortunately it didn't take that long. Sure I can add it to the BreachWarning module, just need to run it through Javier first.